src/admin_module/user_module/membership.py

```diff
@@ -21,12 +23,17 @@ def get_type(self):
         """
         return self._type
 
-    def get_discount(self):
+    def get_discount(self, membership_type):
         """
         Returns the percentage of discount according the membership type
         :return: the percentage from 0 to 100
         """
-        return self._percent_of_discount
+        conn = DBManager()
+        discount = 0
+        execution_query = "select percent_discount from membership where type='" + membership_type + "'"

```

Comment:
It is not recommended to use this type of concatenation try to use "String %s concatenate " % "to"